.travis.yml

```diff
@@ -0,0 +1,18 @@
+before_script:
+  - sh -e /etc/init.d/xvfb start
```

Comment:
Add "export DISPLAY=:99.0" before this line and remove the 'env' section from below. I'd also suggest making this 'before_install' and adding and install section for flake8, and a 'before_script' for running flake8.